ansible/ci-deployment.yml

```diff
@@ -33,8 +47,26 @@
     - role: omero-web-runtime
     - role: nginx
 
+
 # Deploy Documentation prerequisites
 - hosts: ci-docs
   roles:
+    - role: virtualenv
     - role: sphinx-build
     - role: ansible
+
+  vars:
+    virtualenv_user: "{{ jenkinsuser }}"
+    virtualenv_path: /home/{{ jenkinsuser }}/virtualenv
+    virtualenv_site_packages: "yes"
+    virtualenv_pip_rpm_dependencies:
+    virtualenv_pip_dependencies:
```

Comment:
I am not sure of the requirement for a virtualenv in ci-docs groups. As discussed in https://github.com/openmicroscopy/infrastructure/pull/158, the current nature of the documentation build stack (i.e. Ant + Sphinx) makes it not usable with a virtualenv in a Jenkins environment. Is there another use case?